core/templates/dev/head/css/oppia.css

```diff
@@ -2586,6 +2586,7 @@ div#ng-curtain {
 }
 
 .oppia-navbar {
+  top: 0;
```

Comment:
@Showtim3 Add style attributes in sorted order and if possible add this as local styling in the admin page OR make sure to test this thoroughly in all other places. (I would suggest doing the first one to avoid any regression.)

Reply:
Ok DOne